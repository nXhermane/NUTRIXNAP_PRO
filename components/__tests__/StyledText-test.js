import * as React from 'react';
import renderer from 'react-test-renderer';

import { MonoText } from '../StyledText';

describe('MonoText component', () => {
  it('renders "Snapshot test!" correctly', () => {
    const tree = renderer.create(<MonoText>Snapshot test!</MonoText>).toJSON();

    expect(tree).toMatchSnapshot();

    // Add the following line to check if the snapshot is up-to-date
    expect(renderer.create(tree).toJSON()).toMatchSnapshot();
  });
});
