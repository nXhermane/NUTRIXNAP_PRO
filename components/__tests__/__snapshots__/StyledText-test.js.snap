// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<Text
  style={
    [
      {
        "color": {
          "placeholder": "#AFB2BF",
          "primary": "#050404",
          "secondary": "#716969",
        },
      },
      [
        undefined,
        {
          "fontFamily": "SpaceMono",
        },
      ],
    ]
  }
>
  Snapshot test!
</Text>
`;
